src/CalcManager/Ratpack/num.cpp

```diff
@@ -418,12 +418,11 @@ void _divnum(PNUMBER* pa, PNUMBER b, uint32_t radix, int32_t precision)
     }
     destroynum(tmp);
 
-    int32_t digit;
     int32_t cdigits = 0;
     while (cdigits++ < thismax && !zernum(rem))
     {
-        digit = radix - 1;
-        PNUMBER multiple = nullptr;
+        uint32_t digit = radix - 1;
+        PNUMBER multiple = rem;
```

Comment:
Why set this to rem in this case?

Reply:
This is because rem is the starting place before we change the multiple anyway